frontend/src/components/templates/Temp1.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./template1.css";
+import HolderImage from "../assets/choose-image.png";
+
+const Temp1 = () => {
+  const handleImageInput = () => {
```

Comment:
Go through, https://timmousk.com/blog/react-get-element-by-id/ and make the changes accordingly.

Reply:
I have gone through this and I will use useRef hooks in my next PR Today.